recipes/inspector/meta.yaml

```diff
@@ -1,16 +1,23 @@
-{% set version = "1.2" %}
+{% set version = "1.3" %}
+
+channels:
```

Comment:
This is not needed too. 
The defaults channel is paid since recently.

Reply:
Please help clarify. Is only the defaults channel not needed or are all these channels not needed? We found that we cannot install the package via conda if without the channels. We had to use "conda install -c conda-forge -c bioconda -c defaults inspector" to successfully install it.

Reply:
Bioconda builds use only `conda-forge` and `bioconda` channels. It is hardcoded.
AFAIK the `channels` property is not used at all.

Reply:
If you need `-c defaults` then you depend on non-free package.
But the build is green, so I guess you don't really need `defaults`

Reply:
Good to know. I have removed all the channels lines.